en/components/carousel.md

```diff
@@ -14,7 +14,7 @@ Use the Carousel Component to let the user navigate through a collection of imag
 
 ## Styling
```

Comment:
@YoannaIvanova add an image for the section above showing a default slider with the 10 dots and some image that you like :)